docs/conf.py

```diff
@@ -22,7 +22,7 @@
 copyright = u"2019, Sentry Team and Contributors"
 author = u"Sentry Team and Contributors"
 
-release = "0.19.5"
+release = "1.0.0"
```

Comment:
```suggestion
release = "0.19.5"
```
The prepare script does this for us